### PR TITLE
Add manifests

### PR DESCRIPTION
**Pull request by eskombro**
Manifests are generated by Helm. To generate manifests, run `helm template meilisearch charts/meilisearch > manifests/meilisearch.yaml` at the root folder.

Added a `getting started` section in the root README.md to deploy MeiliSearch using either manifests or Helm chart.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,11 +24,58 @@
...
 
-This repository's goal is to collect tools and examples of Kubernetes configurations and packages (Manifests, Helm charts, etc...) in order to make it simple to integrate a MeiliSearch instance in a Kubernetes cluster.
+# Getting started
 
-## HELM
+First of all, you will need a Kubernetes cluster up and running. If you are not familiar with how Kuberentes works or need some help with this step, please check the [Kubernetes documentation](https://kubernetes.io/docs/home/).
+
+## Install kubectl
+
+`kubectl` is the most commonly used CLI to handle a Kubernetes cluster. The installation instructions are [available here](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
```
Comment (tpayet) [inline README.md:33]:
```suggestion
`kubectl` is the most commonly used CLI to manage a Kubernetes cluster. The installation instructions are [available here](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
```

not 100% sure

**Review by tpayet — APPROVED** [review]
Nice :D Should we add a github action that runs `helm template ...` and compare it to the manifest generated? So we can be sure the template is up to date? WDYT?
In any cases, this can be done in another pull request

**Comment by Rio** [comment]
Looking good. It would definitely be nice to automate this in some way. Apart from the manifests itself missing some securityContexts, I'm not really sure it makes sense to generate the helm connection test. Not many tools know how to use a test like that.

**Review by renehernandez — APPROVED** [review]
Looks great

**Pull request merged**